packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -147,6 +147,13 @@ const getUsersElement = (
 ): JSX.Element => {
   const userElements: JSX.Element[] = [];
   typingUsers.forEach((user, index) => {
+    if (user.displayName) {
+      let truncatedDisplayName = user.displayName;
+      if (truncatedDisplayName.length > 50) {
+        truncatedDisplayName = truncatedDisplayName.substring(0, 50) + '...';
+      }
+      user.displayName = truncatedDisplayName;
+    }
```

Comment:
Question: if we assign the truncatedDisplayName back to user.displayName, would that change the message object? If so, would it affect other areas such as sender's name in the message bubble?